appnav/src/test/kotlin/io/element/android/appnav/loggedin/LoggedInPresenterTest.kt

```diff
@@ -60,8 +60,9 @@ class LoggedInPresenterTest {
         }.test {
             val initialState = awaitItem()
             assertThat(initialState.showSyncSpinner).isFalse()
+            roomListService.postSyncIndicator(RoomListService.SyncIndicator.Show)
             consumeItemsUntilPredicate { it.showSyncSpinner }
-            roomListService.postState(RoomListService.State.Running)
+            roomListService.postSyncIndicator(RoomListService.SyncIndicator.Hide)
```

Comment:
`consumeItemsUntilPredicate` now expect the predicate to be true at some point, so I had to fix this outdated test.